src/containers/BookList/BookList.js

```diff
@@ -17,19 +17,21 @@ function BookList({ books }) {
       <tbody>
         {
           books.map(book => (
-            <Book key={book.id} book={book} />
+            <Book key={book.id} book={book} deleteBook={deleteBook} />
           ))
         }
       </tbody>
     </table>
   );
 }
 
+BookList.propTypes = {
+  books: PropTypes.instanceOf(Array).isRequired,
+  deleteBook: PropTypes.func.isRequired,
```

Comment:
- [ ] I got this error console:
![Screenshot from 2020-12-03 19-02-10](https://user-images.githubusercontent.com/4246637/101062263-19dbc100-359a-11eb-999b-3081fb6d89b5.png)

We still missing deleteBook function in this component.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)